src/amber/dsl/router.cr

```diff
@@ -23,4 +23,24 @@ module Amber::DSL
 
     {% end %}
   end
+
+  module ControllerActions
```

Comment:
I'm cool with this all in all. I would really like for us to implement class views eg. For a given `HomeController` you will have a corresponding `HelloView` with this in place we can implement the decorator pattern which allows for greater separation of concerns. An example directory structure would be something like:

Controller
-->HomeController.cr
View
-->templates
-->layouts
-->HomeView.cr